.github/workflows/report_news/main.py

```diff
@@ -58,6 +62,58 @@ def fetch_tech_griot() -> list:
     return articles
 
 
+def fetch_tericcabrel_blog() -> list:
+    """
+    fetching articles from tericcabrel blog
+    """
+
+    with urlopen(TERICCABREL_BLOG_SITEMAP_URL) as response:
+        body = response.read()
+
+    xml_raw = ET.fromstring(body)
+    articles = []
+
+    for url in xml_raw.findall('{http://www.sitemaps.org/schemas/sitemap/0.9}url')[-MAX_ARTICLES:]:
```

Comment:
Feels like restraining it to `MAX_ARTICLES`, would make us skip a bunch of articles published on that blog. Maybe we can have an upper limit of 100 ? The `MAX_ARTICLES` is the maximum amount of article we want to publish on the telegram message, not the max of articles we want to choose from.

Reply:
@rakici 

The sitemap contains a huge amount of articles. Parse all these articles to extract their content will take time and can overload the server.

I opted to take the last n articles.

Reply:
This makes sense howerver, I still want us be drawing from a bigger set of articles than 10, as this can make us skip during an initial run on a blog that has more 10 articles, potential interesting articles.